Misc mappings

mezz opened the pull request.

mezz committed:
Misc mappings

copygirl commented on mappings/net/minecraft/block/state/BlockStateDataArray.mapping:
```diff
@@ -0,0 +1,17 @@
+	METHOD a getSizeBytes ()I
+	METHOD a getBlockState (I)Lnone/asm;
+		ARG 0 blockStateId
+	METHOD a getBlockStateId (Lnone/asm;)I
```
`idToBlockState` and `blockStateToId` ? `get...` sounds like a getter which this isn't?

copygirl commented on mappings/net/minecraft/block/state/BlockStateDataArray.mapping:
```diff
@@ -0,0 +1,17 @@
+	METHOD <init> (ILnone/atu;)V
+		ARG 0 sizeBits
+		ARG 1 expansionDelegate
+	METHOD a getSizeBytes ()I
```
`getSizeInBytes` ?

copygirl commented on mappings/net/minecraft/block/state/BlockStateDataArray.mapping:
```diff
@@ -0,0 +1,17 @@
+CLASS none/atr net/minecraft/block/state/BlockStateDataArray
+	FIELD a states [Lnone/asm;
+	FIELD b expander Lnone/atu;
+	FIELD c sizeBits I
```
`sizeInBits` ?
Though ... maybe `size` or `bits` is good enough?

copygirl commented on mappings/net/minecraft/client/gui/widget/WidgetLock.mapping:
```diff
@@ -0,0 +1,27 @@
+CLASS none/bew net/minecraft/client/gui/widget/WidgetLock
```
~~`ButtonLock`~~ `WidgetLockButton` ?

maxpowa replied:
Should probably be `WidgetButtonLock` to keep in line with `WidgetButtonLanguage`, since they both just render a special texture over the standard button.

copygirl commented on mappings/net/minecraft/entity/player/EntityPlayer.mapping:
```diff
@@ -61,8 +61,11 @@ CLASS none/aam net/minecraft/entity/player/EntityPlayer
 	METHOD a openVillagerGui (Lnone/aih;)V
 	METHOD a openCommandBlockMinecart (Lnone/ail;)V
+	METHOD a setGameMode (Lnone/aiu;)V
+		ARG 0 gameMode
```
`value`

copygirl commented on mappings/net/minecraft/entity/player/PlayerProperties.mapping:
```diff
@@ -0,0 +1,15 @@
+		ARG 0 value
+	METHOD b getGameMode ()Lnone/aiu;
+	METHOD b setGameMode (Lnone/aiu;)V
+		ARG 0 gameMode
```
`value`

maxpowa committed:
Misc random mappings (#120)

* Misc random mappings

* Minor changes & move WorldServerChild to sortme

* username -> userName

copygirl committed:
Rename some Gui related methods & more

lastPressedWidget
mouseDragged => onMouseDragged
handleMouseClick => onMousePressed
handleTouchEvent => onMouseReleased
handleKeyPress => onKeyPressed
handleButtonPressed => onWidgetPressed
WidgetButton.released => onReleased

mezz committed:
Add some mappings for mob pathfinding (#127)

* Add some mappings for mob pathfinding

* Fixes for review

* Fixes for review

Darkhax committed:
Several mappings (#129)

* Mapped a bunch of loot stuff.

* Mapped some random stuff.

* Mapped some debug renderers.

* Mapped a few misc things.

* element -> json

copygirl committed:
Map / rename PlayerFormatUpgrader

copygirl committed:
Fix StatManager, map StatManagerServer and such

updateStatState => set
increaseState => increase
getValue => get
EntityPlayer.getStatManager => getStats

copygirl committed:
Fix LootTable related mappings

serialze => serialize
Serializer => DeSerializer
LootTableContext:
* TARGET => Target
* source => damageSource
* setX(x) => setX(value)

shadowfacts committed:
Random block stuff

shadowfacts committed:
Random world stuff

copygirl committed:
Add some mappings for block rendering (#133)

* Add some mappings for block rendering

* Fixes some things

* Make getAoBrightness to getAmbientOcclusionBrightness

* Fix merge conflicts

* Fix a few minor things

VertexBuffer.putColorRGB => putColor
BlockRenderer hashcode parameter => variationSeed
Revert to IBlockView.getCombinedLight

mezz committed:
BIomes, biome generation (#132)

* BIomes, biome generation

* Misc random mappings (#120)

* Misc random mappings

* Minor changes & move WorldServerChild to sortme

* username -> userName

* Rename some Gui related methods & more

lastPressedWidget
mouseDragged => onMouseDragged
handleMouseClick => onMousePressed
handleTouchEvent => onMouseReleased
handleKeyPress => onKeyPressed
handleButtonPressed => onWidgetPressed
WidgetButton.released => onReleased

* Add some mappings for mob pathfinding (#127)

* Add some mappings for mob pathfinding

* Fixes for review

* Fixes for review

* Several mappings (#129)

* Mapped a bunch of loot stuff.

* Mapped some random stuff.

* Mapped some debug renderers.

* Mapped a few misc things.

* element -> json

* Map / rename PlayerFormatUpgrader

* Fix StatManager, map StatManagerServer and such

updateStatState => set
increaseState => increase
getValue => get
EntityPlayer.getStatManager => getStats

* Fix LootTable related mappings

serialze => serialize
Serializer => DeSerializer
LootTableContext:
* TARGET => Target
* source => damageSource
* setX(x) => setX(value)

* Random block stuff

* Random world stuff

* Add some mappings for block rendering (#133)

* Add some mappings for block rendering

* Fixes some things

* Make getAoBrightness to getAmbientOcclusionBrightness

* Fix merge conflicts

* Fix a few minor things

VertexBuffer.putColorRGB => putColor
BlockRenderer hashcode parameter => variationSeed
Revert to IBlockView.getCombinedLight

* Fix a few things

copygirl committed:
Fix merge conflicts

copygirl committed:
Remove Vector3f (fixes Enigma mapping conflict)

copygirl committed:
Map WidgetLockButton

copygirl committed:
Fix gameMode setters to use "value"

copygirl commented:
Okay.. this didn't go as intended ...
